SeaPublicWebsite/wwwroot/css/_simple-energy-advice-custom.scss

```diff
@@ -381,11 +381,13 @@
     [aria-current] {
         padding: 0.3125em;
     }
+    
+    button {
+        font-size: unset;
+        background: none;
+        border: none;
+        cursor: pointer;
+        color: $govuk-text-colour;
+        padding-inline: 0.3125em;
+    }
 }
-
-.sea-language-select__list-item button {
-    font-size: unset;
```

Comment:
moved these into the parent class for better nesting. I think these are equivalent anyway